roundtables/rails-week3/day5-test-db-flash-session.md

```diff
@@ -0,0 +1,44 @@
+# Exercism: The Test Database, Flash, and Session
+
+Today we are going to be discussing the following concepts in Rails: The Test Database, Flash, and Session
+
+For this discussion, think of your instructor as a mediator and guide, rather than as a lecturer. You are encouraged to do the following:
+
+* Whenever possible, you should use voice.
+* You should pose questions to the group, not the instructor.
+* Similarly, you are encouraged to answer questions, with the instructor available to correct misconceptions and provide answers when no one else has an answer.
+* Say something, even if you think you might be wrong! (Nobody throws a frisbee well on their first try)
+* When discussing code, you are encouraged to use VS Code Liveshare, and or screen sharing through zoom.
+* Take notes!
+
+## Materials
+* The Test Database
+* Flash
+* Session
+* Testing Session
+
+## Prompts
+You are encouraged to copy these questions into a document and take notes during the discussion.
+
+### The Test Database
+
+1. What is the test database, and how is it different from the development database (the database accessed when running the Rails server and visiting `localhost:3000`)?
+1. What is the relationship between fixtures and the test database?
+1. In Media Ranker, how will the test database be used? Use concrete and specific examples when possible.
+
+### Flash
+
+1. As a group, come up with a definition for Flash in your own words
+1. As a group, come up with a definition for Flash.now in your own words. How does it differ from Flash in syntax and use case?
+1. In Media Ranker, how will flash be used?
+1. Share code: If anyone has an example of flash being used in one Controller action (even if it's not working or in-progress), share with the group! Practice talking through your code. Feel free to use this time to ask questions, especially if you don't have flash working.
```

Comment:
Capitalizing `flash` is confusing since the actual syntax is lowercase:

```suggestion
1. As a group, come up with a definition for `flash` in your own words
1. As a group, come up with a definition for `flash.now` in your own words. How does it differ from `flash` in syntax and use case?
1. In Media Ranker, how will `flash` be used?
1. Share code: If anyone has an example of `flash` being used in a controller action (even if it's not working or in-progress), share with the group! Practice talking through your code. Feel free to use this time to ask questions, especially if you don't have `flash` working.
```

Also just tweaked wording a little.